android/src/app/organicmaps/widget/placepage/PlacePageView.java

```diff
@@ -784,6 +785,12 @@ public void onCompassUpdated(double north)
     }
   }
 
+  @Override
+  public void onLocationDisabled() {}
```

Comment:
Should PP hide or "outdate" the already displayed information (distance, arrow, current speed/altitude for My Position) in this case? At least a TODO is needed.